crates/ra_hir/src/nameres/tests/mods.rs

```diff
@@ -433,7 +462,7 @@ fn module_resolution_decl_inside_inline_module_3() {
     "###);
 }
 
-// FIXME: issue #1510. not support out-of-line modules inside inline.
+// FIXME: issue #1529. not support out-of-line modules inside inline.
 #[test]
 #[ignore]
```

Comment:
Can we already unignore some of these tests?

Reply:
In next PR I want add support resolve attribute by parent inline module.
Now such tests are included in the ignored ones